### PR TITLE
Remove one index from log_visit that is actually not needed.

### PR DESCRIPTION
**Pull request by tsteur**
Noticed the index is quite big for `log_visit` table so had a look and noticed there is one index not needed when we move `config_id` to the end of the existing `index_idsite_datetime` index.

We had a quick look and it seems that we query `config_id` always in combination with `visit_last_action_time` so should be all good. Probably in the past this was not the case and therefore this index was needed.

refs #6953 this does not affect `log_link_visit_action` so should be ok?

### CODE REVIEW

**Comment by mattab** [comment]
Before we merge this PR we must decide if #6759 is still valid or not. Maybe we need to wait for 2.13.0

**Comment by mnapoli** [comment]
FYI #6953 is also about `log_visit`

**Comment by tsteur** [comment]
It was changed a few days ago ;) The milestone is already set to 2.13

**Comment by tsteur** [comment]
refs #6759

**Comment by mattab** [comment]
Moving to 3.0.0

**Comment by tsteur** [comment]
Can we merge this one in Piwik 3.0 branch already? If so, I'd create a new PR.

File: core/Db/Schema/Mysql.php [hunk 1]
```diff
@@ -153,8 +153,7 @@ public function getTablesCreateSql()
                               config_id BINARY(8) NOT NULL,
                               location_ip VARBINARY(16) NOT NULL,
                                 PRIMARY KEY(idvisit),
-                                INDEX index_idsite_config_datetime (idsite, config_id, visit_last_action_time),
-                                INDEX index_idsite_datetime (idsite, visit_last_action_time),
+                                INDEX index_idsite_configid_datetime (idsite, visit_last_action_time, config_id),
```
Comment (mattab) [inline core/Db/Schema/Mysql.php]: let's rename the INDEX `index_idsite_datetime_configid` to follow index field order

**Comment by mattab** [comment]
Yes we can issue DB schema change again on Piwik 3.0.0 branch. #6953 was closed

**Comment by tsteur** [comment]
Will issue a new PR

**Pull request closed without merging**